subsys/mpsl/mpsl_init.c

```diff
@@ -54,6 +54,12 @@ static int mpsl_lib_init(struct device *dev)
 		return err;
 	}
 
+	IRQ_DIRECT_CONNECT(TIMER0_IRQn, MPSL_HIGH_IRQ_PRIORITY,
+		    MPSL_IRQ_TIMER0_Handler, 0);
```

Comment:
@sofu1 @rugeGerritsen Should we put ZLI priority on these? `IRQ_ZERO_LATENCY` as last argument.

Reply:
Yes, that was the intention :)